docs/product/session-replay/replay-details.mdx

```diff
@@ -51,3 +51,14 @@ Find a complete list of built-in fields and custom tags associated with a replay
 ## Retention
 
 Replays are retained for 90 days in sentry.io (the same retention period as other event types, such as Errors and Transactions). The retention period is not configurable.
+
+## Inbound Filtering
+
+Session replay supports some inbound filtering rules.  Those rules are:
+
+- IP Address
+- Releases
+- Request URL
+- User-Agent
+
+**Note**: Due to a quirk in how Replay emits successful outcomes versus filtered and dropped outcomes, you may observe a larger than expected increase in filtered outcomes. This is not an error. Filtered outcomes are emitted per **event** whereas successful outcomes are emitted per **replay**.  A replay being a collection of events means there is a natural imbalance in the number of outcomes per category.
```

Comment:
```suggestion
**Note**: Because filtered outcomes are emitted per **segment** whereas successful outcomes are emitted per **replay**  (a replay being a collection of segments), you may see a noticeable increase in filtered outcomes on your [Stats](https://sentry.io/orgredirect/organizations/:orgslug/stats) page. This is not an error.
```
@jas-kas here's what I'd recommend.

Or we can do: **Note**: Because filtered outcomes are emitted per **segment** whereas successful outcomes are emitted per **replay**  (a replay being a collection of segments), you may see significantly more filtered than successful outcomes on your [Stats](https://sentry.io/orgredirect/organizations/:orgslug/stats) page. This is not an error.